types/src/on_chain_config.rs

```diff
@@ -0,0 +1,26 @@
+use crate::{
+    access_path::{AccessPath, Accesses},
+    account_config::{association_address, CORE_CODE_ADDRESS},
+    language_storage::{StructTag, TypeTag},
+};
+use move_core_types::identifier::Identifier;
+
+pub fn access_path_for_config(config_name: Identifier) -> AccessPath {
```

Comment:
can we have some comments explain how this works? it's not intuitive for people not familiar with access_path